flow/core/kernel/power_demand.py

```diff
@@ -0,0 +1,60 @@
+import math
+import random
+import statistics
+import numpy as np
+from scipy.interpolate import interp1d
+from collections import namedtuple
+from flow.energy.base_energy import BaseEnergyModel
+
+class PowerDemandModel(BaseEnergyModel):
+    """Calculate power consumption of a vehicle.
+    Assumes vehicle is an average sized vehicle.
+    The power calculated here is the lower bound of the 
+    actual power consumed by the vehicle
+    """
+
+    def __init__(self,kernel):
+        self.k = kernel
+        self.g = 9.8
+        self.rho_air = 1.225
+        self.Mass = 1200
+        self.rolling_res_coeff = 0.005
+        self.aerodymnamic_drag_coeff = 0.3
+        self.cross_A = 2.6

```

Comment:
- make g = 9.81 to be consistent with some other code we have
- all variable names should be lower case (i.e. `self.mass` and `self.cross_area`)
- typo in 'aerodynamic'
- let's include `self.gamma = 1` here